src/axom/klee/Geometry.hpp

```diff
@@ -69,21 +78,169 @@ class Geometry
+   * \param topology The blueprint topology to use.
+   * \param operator_ a possibly null operator to apply to the geometry.
+   *
+   * \internal TODO: Is this the simplex requirement overly restrictive?
+   */
+  Geometry(const TransformableGeometryProperties &startProperties,
+           const axom::sidre::Group *meshGroup,
+           const std::string &topology,
+           std::shared_ptr<GeometryOperator const> operator_);
+
+  /**
+   * Create a tetrahedron Geometry object.
+   *
+   * \param startProperties the transformable properties before any
+   * operators are applied
+   * \param tet Tetrahedron
+   * \param operator_ a possibly null operator to apply to the geometry.
+   */
+  Geometry(const TransformableGeometryProperties &startProperties,
+           const axom::primal::Tetrahedron<double, 3> &tet,
+           std::shared_ptr<GeometryOperator const> operator_);
+
+  /**
+   * Create a hexahedron Geometry object.
+   *
+   * \param startProperties the transformable properties before any
+   * operators are applied
+   * \param hex Hexahedron
+   * \param operator_ a possibly null operator to apply to the geometry.
+   */
+  Geometry(const TransformableGeometryProperties &startProperties,
+           const axom::primal::Hexahedron<double, 3> &hex,
+           std::shared_ptr<GeometryOperator const> operator_);
+
+  /**
+   * Create a sphere Geometry object.
+   *
+   * \param startProperties the transformable properties before any
+   * operators are applied
+   * \param sphere Analytical sphere specifications
+   * \param levelOfRefinement Number of refinement levels to use for
+   *        discretizing the sphere.
+   * \param operator_ a possibly null operator to apply to the geometry.
+   *
+   * \internal TODO: Is this the simplex requirement overly restrictive?
+   */
+  Geometry(const TransformableGeometryProperties &startProperties,
+           const axom::primal::Sphere<double, 3> &sphere,
+           axom::IndexType levelOfRefinement,
+           std::shared_ptr<GeometryOperator const> operator_);
+
+  /**
+   * Create a volume-of-revolution (VOR) Geometry object.
+   *
+   * \param startProperties the transformable properties before any
+   * operators are applied
+   * \param discreteFunction Discrete function describing the surface
+   *        of revolution.
+   * \param vorBase Coordinates of the base of the VOR.
+   * \param vorDirection VOR axis, in the direction of increasing z.
+   * \param levelOfRefinement Number of refinement levels to use for
+   *        discretizing the sphere.
+   * \param operator_ a possibly null operator to apply to the geometry.
+   *
+   * The \c discreteFunction should be an Nx2 array, interpreted as
+   * (z,r) pairs, where z is the axial distance and r is the radius.
+   * The \c vorBase coordinates corresponds to z=0.
+   * \c vorAxis should point in the direction of increasing z.
+   *
+   * \internal TODO: Is this the simplex requirement overly restrictive?
```

Comment:
```suggestion
   * \internal TODO: Is the simplex requirement overly restrictive?
```

Are comments like this captured in issues so they don't get lost?

Reply:
Deleted the note.